.gitignore

```diff
@@ -19,7 +19,6 @@ xunit_results.xml
 
 # ignore work directories and setup files
 .setup
-.env
```

Comment:
Don't we use this file for local overrides?

Reply:
In this repo - no, this file is not used and there is no code that handles it. Probably it was used in the past, but not now.